src/utils/hooks/useUpdateReviewResponses.ts

```diff
@@ -2,31 +2,55 @@ import {
   ReviewResponse,
   ReviewResponseDecision,
   ReviewResponseRecommendedApplicantVisibility,
+  useUpdateReviewMutation,
   useUpdateReviewResponseMutation,
 } from '../generated/graphql'
 
 // below lines are used to get return type of the function that is returned by useUpdateReviewResponseMutation
 type UseUpdateReviewMutationReturnType = ReturnType<typeof useUpdateReviewResponseMutation>
 type PromiseReturnType = ReturnType<UseUpdateReviewMutationReturnType[0]>
 
-// hook used to update review response, comment and decision, use as per type definition below
-// also computes and updates recommendedApplicantVisibility
-type UseUpdateReviewResponse = () => (
-  reviewResponse: ReviewResponse,
-) => PromiseReturnType
+// Hook to update review responses. Either updates individual response or a
+// batch via updating review object.
+// Also computes recommendedApplicantVisibility
+type UseUpdateReviewResponse = () => {
+  updateReviewResponse: (reviewResponse: ReviewResponse) => PromiseReturnType
+  updateMultipleReviewResponses: any
```

Comment:
Hmm, tbh I am not a fan of 'any', unless it's absolutely necessary, in this case input is know and the return is Promise<void>.

Reply:
Yeah, same. I often put it when I'm trying to test something, and then forget to update it. 🤷‍♂️